tests/test_command_parse.py

```diff
@@ -222,6 +223,10 @@ def test_crawlspider_no_matching_rule(self):
         self.assertRegex(_textmode(out), r"""# Scraped Items  -+\n\[\]""")
         self.assertIn("""Cannot find a rule that matches""", _textmode(stderr))
 
+        status, out, stderr = yield self.execute([self.url('/invalid_url')])
+        self.assertEqual(status, 0)
+        self.assertIn("""""", _textmode(stderr))
```

Comment:
This is checking that there is an empty string somewhere in the stderr string. There is always an empty string in any string. So this line can be removed.